Applications/Spire/Include/Spire/OrderImbalanceIndicator/LocalOrderImbalanceIndicatorModel.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef SPIRE_LOCAL_ORDER_IMBALANCE_INDICATOR_MODEL_HPP
+#define SPIRE_LOCAL_ORDER_IMBALANCE_INDICATOR_MODEL_HPP
+#include <queue>
+#include <unordered_map>
+#include <vector>
+#include "Nexus/Definitions/OrderImbalance.hpp"
```

Comment:
Don't reinclude include headers that are necessarily part of other headers. For example ```OrderImbalanceIndicatorModel.hpp``` necessarily includes ```std::vector```, ```Intervals```, ```QtPromise```, etc... as part of its API, so it should not get included again.